sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryCoderProviderRegistrar.java

```diff
@@ -35,6 +35,8 @@
   public List<CoderProvider> getCoderProviders() {
     return ImmutableList.of(
         CoderProviders.forCoder(TypeDescriptor.of(TableRow.class), TableRowJsonCoder.of()),
-        CoderProviders.forCoder(TypeDescriptor.of(TableRowInfo.class), TableRowInfoCoder.of()));
+        CoderProviders.forCoder(TypeDescriptor.of(TableRowInfo.class), TableRowInfoCoder.of()),
+        CoderProviders.forCoder(
+            TypeDescriptor.of(BigQueryInsertError.class), BigQueryInsertErrorCoder.of()));
```

Comment:
Don't think this is needed - you can just explicitly call setCoder inside BigQueryIO

Reply:
Removed in def18e7
The coder is explicitly assigned [here](https://github.com/apache/beam/pull/5341/files#diff-57a8c3a7f4411b5b5a08ea6017f35b0fR146)